pages/ksphere/kubeflow/1.0.1-0.4.0/download/index.md

```diff
@@ -0,0 +1,12 @@
+---
+layout: layout.pug
+navigationTitle: Download
+title: Download KUDO for Kubeflow
+menuWeight: 1
```

Comment:
Is this really where we want this page to appear in the side bar? (The menuweight determines the order, the lower the value, the 'higher' in the side bar it appears)

As it stands, this commit results in the following list in the left hand side bar:

"Download"
"Introducing KUDO for Kubeflow"
"Legal Notices"
"Manage Secrets"
"Manage Users"
"Tutorials"

I would think we would want "Introducing KUDO for Kubeflow" to be first and then perhaps 'Download' second.   I would think "Legal Notices" would be last.   

Also, it looks like only the first 4 items in the list appear on the main 'ksphere' page, so presumably we don't want to have 'legal notices' as one of the 4 links on the main ksphere page...

Reply:
Good catch! Adjusted, PTAL.

Reply:
Looks much better!